setup.py

```diff
@@ -77,7 +77,7 @@
         'pyyaml',
         'requests',
         'sphinx==1.6.7',
-        'tox',
+        'tox~=3.28.0',
```

Comment:
Given we have bumped `packaging` (https://github.com/Azure/azure-cli/pull/25371), do we still need to pin `Jinja2`?